pkg/rlimit/rlimit.go

```diff
@@ -4,6 +4,6 @@ package rlimit
 // RaiseOpenFilesLimit on Unix raises the current process's soft limit on the
 // number of open files to its hard limit and returns the limit.
 // On other operating systems, RaiseOpenFilesLimit does nothing.
-func RaiseOpenFilesLimit() (uint64, error) {
+func RaiseOpenFilesLimit() (int, error) {
```

Comment:
`syscall.Rlimit.Cur` is `int64` or `uint64` on all platforms and architectures, so I think this should remain `uint64`.